core/backend/src/CheckpointManager.ts

```diff
@@ -307,7 +307,10 @@ export class CheckpointManager {
   private static async doDownload(request: DownloadRequest): Promise<ChangesetId> {
     try {
       // first see if there's a V2 checkpoint available.
+      const stopwatch = new StopWatch(`[${request.checkpoint.changeset.id}]`, true);
+      Logger.logInfo(loggerCategory, `Starting download of V2 checkpoint with id ${stopwatch.description}`);
       const changesetId = await V2CheckpointManager.downloadCheckpoint(request);
+      Logger.logInfo(loggerCategory, `Applied changeset with id ${stopwatch.description} (${stopwatch.elapsedSeconds} seconds)`);
```

Comment:
This should say "Finished download of v2 checkpoint" or something along those lines, no?

Reply:
yes, this message is not correct. At this point you have downloaded a checkpoint and have not applied any changesets. BTW, web backends can NEVER apply changesets (they may only show Named Versoins that must have a checkpoint), so this whole concept of "updating to a version" is only relevant for editable backends.

I see this PR has already merged, but this should be changed.